app/services/idv/inherited_proofing/va/verification_service.rb

```diff
@@ -0,0 +1,25 @@
+module Idv
+  module InheritedProofing
+    module Va
+      class VerificationService
+        # def send_verification_request(applicant)
+        # VerificationRequest.new(config: config, applicant: applicant).send
+
+        # instantiate proofer instance with config
+        # call proof method on proofer with applicant object
+        # handle result
+
+        def initialize(config)
+          @config = config
+        end
+
+        def verify_phone(applicant)
+          # proofer = Proofing::LexisNexis::PhoneFinder::Proofer.new(@config)
+          # result = proofer.proof(applicant)
+          # do something with the result
+          proofer = VerificationRequest.new(config: @config, applicant: applicant).send
+        end
+      end
+    end
+  end
```

Comment:
Is there a way for is to reuse `Idv::Agent.proof_address` instead of making a new class like this? (it's our phone check) https://github.com/18F/identity-idp/blob/main/app/services/idv/agent.rb#L37